extern/qhull/global.c

```diff
@@ -1843,7 +1843,6 @@ void qh_initqhull_start2(FILE *infile, FILE *outfile, FILE *errfile) {
   time_t timedata;
   int seed;
 
-  qh_CPUclock; /* start the clock(for qh_clock).  One-shot. */
```

Comment:
We shouldn't modify anything in `extern/qhull` as it is a verbatim copy of the qhull source code.  I'd prefer to leave it as it is and accept compiler warnings rather than change it.